### DISCUSSION TITLE
CVE-2022-42889 apache.commons.commons-text

### DISCUSSION THREAD

**Discussion by murali-choudari-by** [spotbugs/spotbugs#2232] [Q&A] [answered]
Dear Sir / Madam,

As we know Apache released the fix for Security Vulnerability issue ( CVE-2022-42889 ) for commons.commons-text in 1.10.0 release. is this incorporated in latest spotbugs version if not when are we going to upgrade this specific version for spotbugs ?

Thank you.

Regards,
Murali Choudari.

**Comment by iloveeclipse** [comment, marked as answer]
Spotbugs 4.7.3 includes `commons-text-1.10.0.jar` that corresponds to "patched" `1.10.0` version.

See https://github.com/spotbugs/spotbugs/pull/2197 and https://github.com/spotbugs/spotbugs/releases/tag/4.7.3

**Reply by murali-choudari-by** [reply]
Thanks @iloveeclipse for your quick response.

Regards,
Murali Choudari.

**Comment by hazendaz** [comment]
closing as resolved.
